examples/s3/README.md

```diff
@@ -13,7 +13,7 @@ $ git clone https://github.com/aws/aws-sdk-cpp.git
 $ cd aws-sdk-cpp
 $ mkdir build
 $ cd build
-$ cmake .. -DBUILD_ONLY="s3" \
+$ cmake .. -DBUILD_ONLY="core;s3" \
```

Comment:
pretty sure core is not needed. Which version of the SDK are you using?

Reply:
You're right, something must have been off in my environment because I started with a clean copy and it worked without `core`. I'll remove it.